src/components/ExchangeSearch/index.tsx

```diff
@@ -35,39 +39,42 @@ const ExchangeSearch: React.FC<ExchangeSearchProps> = ({ prices, assets, setAsse
         </div>
       </IonHeader>
       <IonContent className="search-content">
-        <IonList>
-          {assets
-            .filter(
-              (asset: AssetConfig) =>
-                asset.assetHash.toLowerCase().includes(searchString) ||
-                asset.ticker.toLowerCase().includes(searchString) ||
-                asset.coinGeckoID?.toLowerCase().includes(searchString)
-            )
-            .map((asset: AssetConfig, index: number) => {
-              return (
-                <IonItem
-                  className="ion-no-margin"
-                  key={index}
-                  data-asset={index}
-                  onClick={(ev) => {
-                    setAsset(asset);
-                    close(ev);
-                  }}
-                >
-                  <div className="search-item-name">
-                    <span>
-                      <CurrencyIcon currency={asset.ticker} />
-                    </span>
-                    <p>{asset.ticker}</p>
-                  </div>
-                  <div className="search-item-amount">
-                    <span className="price">{(asset.coinGeckoID && prices[asset.coinGeckoID]) || '0'}</span>
-                    <span className="fiat-currency">{currency.toUpperCase()}</span>
-                  </div>
-                </IonItem>
-              );
-            })}
-        </IonList>
+        {assets.length > 0 && (
```

Comment:
what happens if lenght is 0? Shall we show a text?

Reply:
This is how it looks if no assets
<img width="270" alt="Screenshot 2022-02-18 at 12 55 43" src="https://user-images.githubusercontent.com/4022128/154686975-9827de2c-4c5c-4f7b-81eb-47aa833bdecc.png">